tests/common/connections/ssh_console_conn.py

```diff
@@ -15,10 +15,18 @@ def __init__(self, **kwargs):
         self.sonic_username = kwargs['sonic_username']
         self.sonic_password = kwargs['sonic_password']
 
-        if kwargs['console_type'] == CONSOLE_SSH:
+        # Store console type for later use
+        self.console_type = kwargs['console_type']
+
+        if self.console_type == CONSOLE_SSH:
+            # Login requires port to be provided
             kwargs['username'] = kwargs['console_username'] + r':' + str(kwargs['console_port'])
             self.menu_port = None
+        elif self.console_type == CONSOLE_SSH_DIGI_CONFIG:
```

Comment:
So I use that console type only here (instead of within the csv file): 
https://github.com/sonic-net/sonic-mgmt/blob/fc38a83f79c3a080ede5a1024bb991e590693d0a/tests/conftest.py#L1793

I didn't want to make changes to the csv files because the way they connect on the happy path is the same as before (a `CONSOLE_SSH` connection) - it's only in the rare case that this issue occurs that we need to first log in with the alternative approach, before again retrying setup.

I introduced that connection type specifically for use within the `duthost_clear_console_port` function (only used to recover from that exception).

If, in future, we need to use that connection type in more areas, it might be possible to have a new csv file for those usecases? Not 100% sure, but for now we have enough information from the initial connection attempt to establish this connection type, so that's how I approached it - happy to change if we think we should though.